net/scripts/install-redis.sh

```diff
@@ -8,3 +8,7 @@ add-apt-repository -y ppa:chris-lea/redis-server
 apt-get --assume-yes install redis
 
 systemctl enable redis-server.service
+
+sed -i '/^maxmemory .*/ s//maxmemory 8gb # limit set by solana\/net\/scripts\/install-redis.sh/' /etc/redis/redis.conf
```

Comment:
Would redis barf if we did this the braindead way?
```bash
cat "maxmemory 8gb" >> /etc/redis/redis.conf
```
I'm worried about the case of a redis.conf that doesn't contain `/^maxmemory .*/`

Reply:
maybe do a bash `if grep` block here? I don't know it offhand, but if you do, feel free to share -- if not, I can look it up!

Reply:
(I'll do the same for memory eviction -- it's also set in the default redis config but if we're assuming an unfriendly config, I'll take care of it for both)

Reply:
Not sure off hand.  An `if grep` would work too but is more code than the braindead way (if it works!)

Reply:
ok I'll try to wrangle up something like this:

![image](https://user-images.githubusercontent.com/153621/68727626-f3b04100-0592-11ea-8b6b-343a8e9e6dad.png)